crates/interfaces/src/executor.rs

```diff
@@ -9,6 +9,8 @@ pub enum BlockValidationError {
     EVM { hash: H256, message: String },
     #[error("Failed to recover sender for transaction")]
     SenderRecoveryError,
+    #[error("Incrementing balance failed")]
+    IncrementBalanceFailed,
```

Comment:
when does this happen?

Reply:
When we fail to increment the balance with withdrawals/block rewards etc.